bot/resources/tags/equals-true.md

```diff
@@ -0,0 +1,24 @@
+---
+embed:
+    title: "Comparisons to `True` and `False`"
+---
+It's tempting to think that if statements always need a comparison operator like `==` or `!=`, but this isn't true.
+If you're just checking if a value is truthy or falsey, you don't need `== True` or `== False`.
```

Comment:
> truthy or falsey

Should we expand on what these mean?

Reply:
I lean towards no. It would make the scope of the tag a lot bigger and is not necessarily relevant to the reader. As is, newbies can make an assumption about what they mean and still understand the rest or they can ask what they mean. Either are good outcomes.